### PR TITLE
label and focus of multiselect fixed

### PR DESCRIPTION
**Pull request by GustavoUT22**

### CODE REVIEW

File: src/types/ui/Multiselect.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 export interface InputBoxProps {
```
Comment (Mardecera) [inline src/types/ui/Multiselect.ts:1]: Missing "I" from interface.

File: src/ui/MultiSelect/MultiSelect.styles.ts [hunk 2]
```diff
@@ -1,31 +1,82 @@
...
   display: flex;
   flex-direction: column;
   width: 100%;
-  background-color: ${colors.gray[150]};
+  background-color: ${colors.tertiary.normal[50]};
   padding: 1.5rem 1.75rem;
   border: ${({ $isOpen }) =>
-    $isOpen &&
-    `0.1rem solid ${colors.primary.main}`}; /* change de color with the correct palette */
+    $isOpen && `0.1rem solid ${colors.tertiary.focus[500]}`};
```
Comment (Mardecera) [inline src/ui/MultiSelect/MultiSelect.styles.ts:57]: Border of `0.125rem `

File: src/ui/MultiSelect/MultiSelect.styles.ts [hunk 3]
```diff
@@ -1,31 +1,82 @@
...
+        gap: 0;
+      }
+      .span-question {
+        animation: ${textAnimation} 0.3s ease forwards;
+      }
+
+      .arrow-button {
+        animation: ${dropAnimation} 0.3s ease forwards;
+      }
+    `};
```
Comment (Mardecera) [inline src/ui/MultiSelect/MultiSelect.styles.ts:79]: Animations or transitions? The animations don't work for me.

File: src/ui/MultiSelect/MultiSelect.styles.ts [hunk 4]
```diff
@@ -89,46 +139,54 @@ export const DisplayOptions = styled.div<InputBoxProps>`
   z-index: 2;
   width: 100%;
   border: ${({ $isOpen }) =>
-    $isOpen &&
-    `0.1rem solid ${colors.primary.main}`}; /* change de color with the correct palette */
+    $isOpen && `0.1rem solid ${colors.tertiary.focus[500]}`};
```
Comment (Mardecera) [inline src/ui/MultiSelect/MultiSelect.styles.ts:142]: border of `0.125rem`

File: src/ui/SearchMultiSelect/SearchMultiSelect.styles.ts [hunk 5]
```diff
@@ -5,10 +5,10 @@ import { colors } from 'styles/colors'
 export const InputSearchWrapper = styled.div`
   display: flex;
   align-items: center;
-  background-color: ${colors.gray[150]};
+  background-color: ${colors.tertiary.normal[50]};
```
Comment (Mardecera) [inline src/ui/SearchMultiSelect/SearchMultiSelect.styles.ts:8]: BG tertiary 200

**Review by Mardecera — CHANGES_REQUESTED** [review]
1. Input multiselect keeps jumping when you click to see its options.
2. When there are no matches in the inputSearch the lower borders on the left and right side do not appear
Captura de pantalla 2024-05-17 092012

File: src/ui/MultiSelect/MultiSelect.styles.ts (hunk 2 above)
Comment (GustavoUT22) [reply src/ui/MultiSelect/MultiSelect.styles.ts:57]: done

File: src/ui/MultiSelect/MultiSelect.styles.ts (hunk 4 above)
Comment (GustavoUT22) [reply src/ui/MultiSelect/MultiSelect.styles.ts:142]: done

File: src/ui/SearchMultiSelect/SearchMultiSelect.styles.ts (hunk 5 above)
Comment (GustavoUT22) [reply src/ui/SearchMultiSelect/SearchMultiSelect.styles.ts:8]: done

File: src/types/ui/Multiselect.ts (hunk 1 above)
Comment (GustavoUT22) [reply src/types/ui/Multiselect.ts:1]: done

**Review by Mardecera — APPROVED** [review]
Nice!

**Pull request merged**